services-turf/src/main/java/com/mapbox/turf/TurfMeasurement.java

```diff
@@ -395,4 +396,46 @@ public static Point along(@NonNull LineString line, @FloatRange(from = 0) double
     }
     return bbox;
   }
+
+  /**
+   * Takes a {@link MultiPolygon} and measures each polygons perimeter in the specified units. if
+   * one of the polygons contains holes, the perimeter will also be included.
+   *
+   * @param multiPolygon geometry to measure
+   * @param units        one of the units found inside {@link TurfConstants.TurfUnitCriteria}
+   * @return total perimeter of the input polygons combined, in the units specified
+   * @see <a href="http://turfjs.org/docs/#linedistance">Turf Line Distance documentation</a>
+   * @since 1.2.0
+   */
+
+  /**
+   * Takes a {@link BoundingBox} and uses its coordinates to create a {@link Polygon}
+   * geometry.
+   *
+   * @param boundingBox a {@link BoundingBox} object to calculate with
+   * @return a {@link Polygon} object
+   * @see <a href="http://turfjs.org/docs/#bboxPolygon">Turf BoundingBox Polygon documentation</a>
+   * @since 4.7.0
+   */
+  public static Polygon bboxPolygon(BoundingBox boundingBox) {
+    double west = boundingBox.west();
+    double south = boundingBox.south();
+    double east = boundingBox.east();
+    double north = boundingBox.north();
+
+    double[] lowLeft = new double[]{south,west};
+    double[] topLeft = new double[]{north,west};
+    double[] topRight = new double[]{north,east};
+    double[] lowRight = new double[]{south,east};
+
```

Comment:
do you really need these double[] arrays? I think you can just use `west, south, east, north` .

Reply:
Don't need them. Originally had them to match the turf.js version for continuity. Switched to your suggestion of `boundingBox.north()`, `boundingBox.east()`, etc.

Just pushed and rebased